.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           name: Install python dependencies
           command: |
             . venv/bin/activate
-            python -m pip install -r requirements.txt
+            python -m pip install -r requirements.txt -c "napari/resources/constraints/constraints_py3.10_docs.txt"
```

Comment:
We need to update this constraint first, though: https://github.com/napari/napari/blob/467d57b3967074370b2ad6750bdfc1af8aee67d6/resources/constraints/constraints_py3.10_docs.txt#L507

Reply:
Why?

Reply:
The import failure in https://github.com/napari/docs/issues/259 is due to sphinx < 5
(That import requires sphinx >5.1)
https://github.com/napari/docs/pull/256#issuecomment-1786315010

Reply:
Oh, nvm, I see. The constraints pins back sphinx_tags:
https://github.com/napari/napari/blob/467d57b3967074370b2ad6750bdfc1af8aee67d6/resources/constraints/constraints_py3.10_docs.txt#L539

Reply:
Yes. constraints pins `sphinx-tags` and `napari-sphinx-theme`. 

And until update constraints files there will be no problem.  

Of course, relax sphinx in requirements.txt is required for constraints compilation provide newer sphinx and `napari-sphinx-theme`